compiler+runtime/include/cpp/jank/runtime/ns.hpp

```diff
@@ -50,6 +50,8 @@ namespace jank::runtime
     /* TODO: Benchmark the use of atomics here. That's what Clojure uses. */
     folly::Synchronized<obj::persistent_hash_map_ptr> vars;
     folly::Synchronized<obj::persistent_hash_map_ptr> aliases;
+
+    std::atomic_uint64_t symbol_counter{};
```

Comment:
Any reason you switched from `atomic_size_t` to `atomic_uint64_t`?

Reply:
I couldn't figure out how to generate IR for `size_t` while calling the `jank_set_ns_symbol_counter`.

For `atomic_uint64_t` I could use the `ctx->builder->getInt64Ty()`. But couldn't find an equivalent for size_t.

Would appreciate if you could point me to the correct place.

Reply:
Ah, I see. The IR is what led to this. Ok, we can keep it as is in order to be a better 1:1 fit with the IR.